tests/Constraint/MultiConstraintTest.php

```diff
@@ -83,6 +84,7 @@ public function testMultiVersionMatchFails()
         $multiRequire = new MultiConstraint(array($this->versionRequireStart, $this->versionRequireEnd));
 
         $this->assertFalse($multiRequire->matches($versionProvide));
+        $this->assertFalse($this->matchCompiled($multiRequire, '==', 1.2));
```

Comment:
The same kind of assertions should be added in all tests.

Reply:
This should be the case. 
I only let non supported matches: `Multi->match(Multi)`. If I missed one please tell me.